code/modules/mob/living/carbon/human/death.dm

```diff
@@ -75,14 +75,22 @@
 	// Finding the last guy for anti-delay.
 	if(SSticker.mode && SSticker.mode.is_in_endgame && SSticker.current_state != GAME_STATE_FINISHED && is_mainship_level(z))
 		var/mob/last_living_human
+		var/shipside_humans_count = 0
+		var/datum/hive_status/main_hive = GLOB.hive_datum[XENO_HIVE_NORMAL]
+		var/see_humans_on_tacmap = main_hive.see_humans_on_tacmap
 		for(var/mob/living/carbon/human/cur_human as anything in GLOB.alive_human_list)
 			if(!is_mainship_level(cur_human.z))
 				continue
-			if(last_living_human)
+			shipside_humans_count++
+			if(last_living_human && see_humans_on_tacmap)
 				last_living_human = null
 				break
 			last_living_human = cur_human
-		if(last_living_human && (GLOB.last_qm_callout + 2 MINUTES) < world.time)
+
+		if(!see_humans_on_tacmap && shipside_humans_count < main_hive.totalXenos.len * 0.75)
```

Comment:
This means they would need to outnumber them by 33%, doesn't it?
For example as it currently is, 12 xenos needs less than 9 humans remaining. 12/9 = 133%

If you wanted 25 percent, you'd need to place it like this on the opposite side
```suggestion
		if(!see_humans_on_tacmap && shipside_humans_count * 1.25 < main_hive.totalXenos.len)
```
or adjust it like this so that its the reciprocal of 1.25
```suggestion
		if(!see_humans_on_tacmap && shipside_humans_count < main_hive.totalXenos.len * 0.8)
```

eg. 15 xenos requires less than 12 humans, 15/12 = 125%
This may seem counterintuitive at first, but 1.25 (5/4) and 0.8 (4/5) are reciprocals, so 1.25 * 0.8 = 1.

Reply:
:nerd_face: